Sources/ProjectDescription/Target.swift

```diff
@@ -104,7 +109,8 @@ public struct Target: Codable, Equatable {
                 dependencies: [TargetDependency] = [],
                 settings: Settings? = nil,
                 coreDataModels: [CoreDataModel] = [],
-                environment: [String: String] = [:]) {
+                environment: [String: String] = [:],
+                modulemap: Path? = nil) {
```

Comment:
minor:

What is everyones thoughts on moving it to somewhere in the middle (e.g. after entitlements)? 

Thinking of how it appears in usage?

```swift
let target = Target(
    name: "MyTarget",
    dependencies: [
    // ..
    ],
     modulemap: "path/to/module.modulemap")
```

```swift
let target = Target(
    name: "MyTarget",
    modulemap: "path/to/module.modulemap",
    dependencies: [
    // ..
    ])
```